schemas/org.gnome.shell.extensions.pop-shell.gschema.xml

```diff
@@ -173,5 +173,11 @@
             <default><![CDATA[['<Primary>Right','<Primary>l']]]></default>
             <summary>Swap window right</summary>
         </key>
+
+        <!-- Launcher Options -->
+        <key type="s" name="search-engine">
+            <default>"https://www.google.com/search?q="</default>
```

Comment:
Would something like duckduckgo be better for a default?

Reply:
It would indeed